tests/data/plpgsql_query.json

```diff
@@ -0,0 +1,91 @@
+[
+  {
+    "PLpgSQL_function": {
+      "action": {
+        "PLpgSQL_stmt_block": {
+          "body": [
+            {
+              "PLpgSQL_stmt_execsql": {
+                "into": true,
+                "lineno": 5,
+                "sqlstmt": {
+                  "PLpgSQL_expr": {
+                    "query": "SELECT details FROM t             WHERE col = input"
+                  }
+                },
+                "target": {
+                  "PLpgSQL_row": {
+                    "fields": [
+                      {
+                        "name": "result",
+                        "varno": 2
+                      }
+                    ],
+                    "lineno": 5,
+                    "refname": "(unnamed row)"
+                  }
+                }
+              }
+            },
+            {
+              "PLpgSQL_stmt_return": {
+                "expr": {
+                  "PLpgSQL_expr": {
+                    "query": "result"
+                  }
+                },
+                "lineno": 6
+              }
+            }
+          ],
+          "lineno": 4
+        }
+      },
+      "datums": [
+        {
+          "PLpgSQL_var": {
+            "datatype": {
+              "PLpgSQL_type": {
+                "typname": "UNKNOWN"
+              }
+            },
+            "refname": "input"
```

Comment:
It's surprising that it couldn't find the data type for `input`, since it's clearly listed in the function arguments.

Reply:
I think that might be a limitation of the libpg_query logic here - we could investigate and see if we can fix that, if helpful.

Reply:
It's not strictly needed for what we're looking at internally; it was just surprising.